main.go

```diff
@@ -39,35 +44,40 @@ func getOpenAPIHandler() http.Handler {
 func main() {
 	useTLS := strings.ToLower(os.Getenv("SERVE_HTTP")) != "true"
 
-	log := grpclog.NewLoggerV2(os.Stdout, ioutil.Discard, ioutil.Discard)
-	grpclog.SetLoggerV2(log)
+	logger := initLogger()
```

Comment:
To be honest I do not like var name logger. As for me `log` is better.

Reply:
but it can be misleading, since we have a std `log` package.

Reply:
But we do not use it

Reply:
I prefer `logger`, because of the hypothetical misleading issue